test/utests/tests/FragmentCollectorMpdTests/FragmentCollectorMpdTestCases.cpp

```diff
@@ -0,0 +1,165 @@
+/*
+ * If not stated otherwise in this file or this component's license file the
+ * following copyrights and licenses apply:
```

Comment:
Sorry, late getting to this as I was travelling.  I would prefer if you used the regular RDK header going forward - It says "copyright", not "copyrights".  The tools will complain for the latter.  It would be better not to have special handling in aamp only.  Thank you!